src/main/java/org/apache/maven/wrapper/DefaultDownloader.java

```diff
@@ -93,6 +99,54 @@ private void downloadInternal(URI address, File destination) throws Exception {
     }
   }
 
+  private void addBasicAuthentication(URI address, URLConnection connection) throws IOException {
+    String userInfo = calculateUserInfo(address);
+    if (userInfo == null) {
+        return;
+    }
+    if (!"https".equals(address.getScheme())) {
+       Logger.info("WARNING Using HTTP Basic Authentication over an insecure connection to download the Maven distribution. Please consider using HTTPS.");
```

Comment:
It looks a bit confusing to put a WARNING into info logging. Why not use warn logging?